--- /opt/ros/electric/stacks/perception_pcl/pcl/include/pcl-1.1/pcl/registration/impl/correspondence_types.hpp	2011-11-17 17:21:26.000000000 +0100
+++ correspondence_types.hpp	2012-07-13 15:46:39.657054489 +0200
@@ -68,9 +68,9 @@
       }
       /** \brief Ctor */
       inline Correspondence(int _indexQuery, int _indexMatch, float _distance)
-      : indexQuery (indexQuery)
-      , indexMatch (indexMatch)
-      , distance (distance)
+      : indexQuery (_indexQuery)
+      , indexMatch (_indexMatch)
+      , distance (_distance)
       { 
       	data[3] = 1.0f;
       }
